docs/naps/2-conda-based-packaging.md

```diff
@@ -0,0 +1,557 @@
+* Allow for simultaneous install of plugins. Currently multiple plugins can be selected for
+  install, uninstall and update, but each on of these actions, are queued and run sequentially.
+* Allow advanced users to install packages from PyPI or other local sources via drag-and-drop. This
+  presents many risks that could irreversibly disrupt the installation, though, so it needs to be
+  considered very carefully.
+
+### Milestone 4: Enabling in-app napari version updates
+
+Our first attempts [^in-app-update-pr] to implement this consisted of a pop-up dialog that would
+notify the user about the available updates. Accepting the dialog would trigger the creation of a
+new environment (e.g. `napari-0.4.16`) alongside the existing one (e.g. `napari-0.4.15`). While it
+could be argued that this is not an "update" per se, but a new installation, it fulfills the same
+goal: the new version is available. If the user wants to get rid of the old one, an option can
+be offered to auto-clean on success, or after not using it for some configurable time.
+
+However, having several versions of napari coexisting might risk questions like "which napari
+version is the default one?", "which one will be in charge of updating napari?", "what if the
+update logic changes over time?". There possible workarounds, but from the user perspective it could
+get confusing very fast.
+
+For that reason, we decided to slightly redesign our approach to in-app updates. We will still use
+several environments to keep things tidy and performant (see "Detailed description" for Milestone 2
+for more details), but the update checks will mimic something closer to a local server-client model
+or, more accurately, a `napari-updater` process (name subject to change; other options include
+`napari-launcher` or  `napari-manager`) that can be queried by a running `napari` instance.
+
+* `napari-updater`: We will create a separate standalone package to handle updates outside of the
+  `napari` process. This will be designed in an application agnostic way while targeting the needs
+  of the napari project. It will allow the user to handle updates but also manage the existing
+  napari installations (investigate issues, remove old ones, clean expired caches, etc).
+* `napari` itself: Each napari version will have a very basic notion about `napari-updater` and
+  will query for updates every time it starts, and also periodically while running. If one is
+  found, the server will run the update on its own. In essence, all `napari` has to do is run
+  `napari-updater` on a separate, detached process every now and then.
+
+The governance of `napari-updater` part will still fall under the `napari` organization (much like
+`magicgui`) but it will be usable outside of the napari project, following the same philosophy we
+adopted for the constructor work.
+
+### Milestone 5: Deprecating Briefcase-based installers
+
+See the corresponding section in "Detailed description".
```

Comment:
this is recursive?

Reply:
Not recursive, but maybe a bit confusing :D I am referring to this paragraph in the preceding section (this subsection is under "Implementation")

> Once we are satisfied with how `conda` installers are working for the end user, we can think of
removing the pipelines that build the Briefcase-based installers. Ideally, we mark them for
deprecation on one release, and then completely remove them on the next one. This involves
deleting the build scripts, the CI workflows, some metadata in `setup.cfg` and possibly some
utility functions and code paths in napari itself.